### PR TITLE
Implement plotting KiCad schematic to SVG

### PR DESCRIPTION
**Pull request by kasbah**
Well, this all a bit scary. This plots KiCad schematics by running an X server (xvfb) inside docker (to allow opening multiple instances),  :sparkles: _operating the GUI via xdotool_ :sparkles: (ಠ_ಠ) -- and all _inside_ the processor docker container (they are actually all the same docker daemon because we mount `docker.sock` into the processor). It does seem to work though and there currently doesn't seem any other way to plot KiCad schematics.

### CODE REVIEW

**Comment by AbdulrhmnGhanem** [comment]
> Well, this all a bit scary. 

Yes :laughing:

**Comment by AbdulrhmnGhanem** [comment]
Is it ready for review?

**Comment by kasbah** [comment]
Yep


On July 19, 2021 at 9:23 PM, Ghanem ***@***.***) wrote:
Is it ready for review?

File: processor/test/integration/test_projects.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const exec = util.promisify(cp.exec)
 
 const { createApp } = require('../../src/app')
 
-const tmpDir = '/tmp/kitspace-processor-test-from-folder'
+const tmpDir = '/data/test/temp/kitspace-processor-test-from-folder'
```
Comment (AbdulrhmnGhanem) [inline processor/test/integration/test_projects.js:11]: Will this path be treated as an actual temp path?

**Review by AbdulrhmnGhanem — APPROVED** [review]

File: processor/test/integration/test_projects.js (hunk 1 above)
Comment (kasbah) [reply processor/test/integration/test_projects.js:11]: Kind of, I added a `posttest` script that removes the whole `/data/test` directory.

**Pull request merged**